benchexec/test_core_assignment.py

```diff
@@ -21,14 +24,21 @@ def lrange(start, end):
 
 
 class TestCpuCoresPerRun(unittest.TestCase):
+    num_of_cores = 0
+    num_of_packages = 0
+    num_of_groups = 0
+    num_of_NUMAs = 0
+    num_of_L3_regions = 0
+    num_of_hyperthreading_siblings = 0
```

Comment:
I think that having the the hyperthreading_siblings per core is better for interpretation, because we also test 3 HT siblings per core. I could of course rename the variable, if you prefer that. 

Groups is just another topology layer, the name is lend from lstopo.

None vs 0 is done in commit https://github.com/sosy-lab/benchexec/pull/892/commits/0689df517323f46784726a9fc5b3d74c0abcdad0